cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -150,15 +158,13 @@ public Boolean call() throws Exception {
 
 
   private void checkServicesWithRetry(Callable<Boolean> callable,
-                                      String exceptionMessage) throws TimeoutException {
-    int numSecs = 0;
+                                      String exceptionMessage) throws TimeoutException, InterruptedException {
```

Comment:
well, but now we throw the exception if sleep is interrupted, instead of continuing with retry, that's not ideal right?